---
float32[] tip_pose